content/en/blog/2025/otel-arrow-phase-2.md

```diff
@@ -0,0 +1,67 @@
+---
+title: OpenTelemetry Protocol with Apache Arrow - Phase 2 Announcement
+linkTitle: OTel-Arrow Phase 2
+date: 2025-05-12
```

Comment:
I am open to this week, but next week seems better at this point.

Reply:
Sure, we can do that! I'll get the CI passing and update the date. Thanks!